pkgs/misc/urbit/fetch-github-lfs.nix

```
@@ -0,0 +1,107 @@
+{ lib, stdenvNoCC, runCommandLocal, cacert, curl, jq }:
+
+{ src
+# `name` shouldn't use `baseNameOf` otherwise we'll
+# get `is not allowed to refer to a store path` errors.
+, name ? baseNameOf src, owner ? "urbit", repo ? "urbit"
+, preferLocalBuild ? true }:
```

```suggestion
}:
```
There is no reason to enforce this here